perception/autoware_traffic_light_visualization/src/traffic_light_roi_visualizer/node.cpp

```diff
@@ -31,7 +31,8 @@ TrafficLightRoiVisualizerNode::TrafficLightRoiVisualizerNode(const rclcpp::NodeO
   using std::placeholders::_2;
   using std::placeholders::_3;
   using std::placeholders::_4;
-  enable_fine_detection_ = this->declare_parameter<bool>("enable_fine_detection");
+  enable_fine_detection_ = this->declare_parameter<bool>("enable_fine_detection", false);
+  use_image_transport_ = this->declare_parameter<bool>("use_image_transport", false);
```

Comment:
By the [autoware coding guideline](https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/parameters/#workflow), parameters will be declared without default values.

Hence,

```suggestion
  enable_fine_detection_ = this->declare_parameter<bool>("enable_fine_detection");
  use_image_transport_ = this->declare_parameter<bool>("use_image_transport");
```

Reply:
Thank you. Sorry, I forgot the determined guideline on parameters. I have pushed a new commit and see if the CI works.